pandas/core/strings/accessor.py

```diff
@@ -2210,26 +2210,66 @@ def removesuffix(self, suffix: str):
         return self._wrap_result(result)
 
     @forbid_nonstring_types(["bytes"])
-    def wrap(self, width: int, **kwargs):
+    def wrap(
+        self,
+        width: int = 70,
```

Comment:
This should remain positional by default

Reply:
Thanks. pushed a fix for this!